test/action.yaml

```diff
@@ -25,6 +25,16 @@ runs:
     - name: Checkout
       uses: actions/checkout@v3
       if: inputs.checkout-repo
+    - name: Verify needed scripts exist
+      shell: bash
+      run: |
+        if [[ -x ./script/setup && -x ./script/test ]]; then
+          echo "Needed scripts do exist and are executable"
+        else
+          echo "Needed scripts do NOT exist or are not executable"
```

Comment:
This should probably use the GHA message formatter `::error::` which bubbles up out of the logs as a banner on the Checks tab.

Reply:
Ah ok, will do. I still have yet to test these changes, but will get to that later today.